lib/dal/src/schema/variant.rs

```diff
@@ -1136,16 +1121,15 @@ impl SchemaVariant {
         schema_variant_id: SchemaVariantId,
     ) -> SchemaVariantResult<PropNodeWeight> {
         let workspace_snapshot = ctx.workspace_snapshot()?;
-        let edge_targets: Vec<NodeIndex> = workspace_snapshot
+        let edge_targets: Vec<_> = workspace_snapshot
             .edges_directed(schema_variant_id, Direction::Outgoing)
             .await?
             .into_iter()
-            .map(|(_, _, target_idx)| target_idx)
+            .map(|(_, _, target_id)| target_id)
             .collect();
```

Comment:
Just noticing while I'm in here: we don't need to do a collect() here (and shouldn't). Not something you need to fix.